elixir/apps/web/lib/web/controllers/auth_controller.ex

```diff
@@ -61,6 +61,14 @@ defmodule Web.AuthController do
     end
   end
 
+  def verify_credentials(conn, %{"account_id_or_slug" => account_id_or_slug} = params) do
+    Logger.warning("Invalid request parameters", params: params)
+
+    conn
+    |> put_flash(:error, "Invalid request parameters: #{inspect(params)}")
```

Comment:
Do you think rendering the URL params here could cause a security issue? We could omit them, but then we lose potentially helpful error codes.

Reply:
From a "can external people exploit this" perspective, I don't think there's a huge risk here, but from a "are we displaying sensitive info on the screen" perspective, this could potentially do that.

Aside from the security aspect, what is this function used for?  From what I can tell we only call the `verify_credentials` function on the `:userpass` provider, so when would this function be called?

Reply:
Yeah, I can definitely understand the desire for getting those helpful errors back to the end user, but I think there is a small possibility of printing sensitive info on the screen if we just dump everything.  Again, it may not be the biggest security risk in the world, but if there is a way we can more selectively print the info I think that would be good.

Reply:
Updated